plan/logical_test.go

```diff
@@ -165,14 +165,19 @@ func (MergeFiltersRule) Pattern() plan.Pattern {
 			plan.Any()))
 }
 
-func (MergeFiltersRule) Rewrite(pn plan.PlanNode) (plan.PlanNode, bool) {
+func (MergeFiltersRule) Rewrite(pn plan.PlanNode) (plan.PlanNode, bool, error) {
```

Comment:
This doesn't need to happen now, but we should probably define this with filter.

Reply:
No that's fine. It makes sense.